src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected String description;
+    private boolean isComplete;
+
+    public Task(String description) {
+        this.description = description;
+        this.isComplete = false;
+    }
+
+    @Override
+    public String toString() {
+        //Subject to change; return string rep for printing purposes
+        return this.getCompletionStatusIcon() + " " + this.description; 
+    }
+
+    protected boolean markAsComplete() {
+        this.isComplete = true;
+        return this.isComplete;
+    }
+
+    protected boolean checkIsComplete() {
+        return this.isComplete;
+    }
+
+    private String getCompletionStatusIcon() {
+        return (isComplete ? "[\u2713]" : "[\u2718]");
+    }
```

Comment:
Should you avoid using ```this``` if it is unnecessary, aka when it is clear that the variable is referring to the class element? It will also be good to keep it consistent as you used ```this.isComplete``` in ```checkIsComplete()``` but only ```isComplete``` in ```getCompletionStatusIcon```.  Other than this, well done!